auth/basics/js.md

```diff
@@ -93,15 +93,15 @@ skygear.auth.signupWithUsername(username, password).then((user) => {
   console.log(user); // user object
 }, (error) => {
   console.error(error);
-  if (error.error.code === skygearError.Duplicated) {
+  if (error.error.code === skygear.ErrorCodes.Duplicated) {
```

Comment:
You may just take away `import skygearError from 'skygear/lib/error';`.

Because `skygearError` is no longer used.